### PR TITLE
VB-785: Send SMS confirmation

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

**Review by tpmcgowan — DISMISSED** [review]
LGTM!

**Comment by psoleckimoj** [comment]
@tpmcgowan made a proper PR, not ready for merging just yet as need a confirmation from Phoebe re merging as Notify service is trial and I can't create a live key.

File: server/config.ts [hunk 1]
```diff
@@ -128,6 +128,13 @@ export default {
       },
       agent: new AgentConfig(Number(get('WHEREABOUTS_API_TIMEOUT_RESPONSE', 10000))),
     },
+    notifications: {
+      enabled: get('SMS_NOTIFICATIONS_ENABLED', 'false', requiredInProduction) === 'true',
```
Comment (psoleckimoj) [inline server/config.ts:132]: This ensure a boolean value

File: server/routes/bookAVisit.ts [hunk 2]
```diff
@@ -467,6 +471,22 @@ export default function routes(
       })
     }
 
+    if (config.apis.notifications.enabled) {
```
Comment (psoleckimoj) [inline server/routes/bookAVisit.ts:474]: Added a switch, just in case

File: server/routes/bookAVisit.ts [hunk 3]
```diff
@@ -467,6 +471,22 @@ export default function routes(
...
+      try {
+        await notificationsService.sendSms({
+          phoneNumber: visitSessionData.mainContact.phoneNumber,
```
Comment (tpmcgowan) [inline server/routes/bookAVisit.ts:477]: Just wondering if it's worth cleaning up the phone number in case spaces in it are an issue...?

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Review by tpmcgowan — APPROVED** [review]
👍

**Pull request merged**